acquisitions/src/main/resources/thunderjet/mod-orders/features/close-order-when-fully-paid-and-received.feature

```diff
@@ -1,84 +1,115 @@
-@parallel=false,
 Feature: Verify once poline fully paid and received order should be closed
 
   Background:
-    * print karate.info.scenarioName
-
     * url baseUrl
-    # uncomment below line for development
-#    * callonce dev {tenant: 'testorders'}
     * callonce login testAdmin
     * def okapitokenAdmin = okapitoken
-    * print okapitokenAdmin
-
     * callonce login testUser
     * def okapitokenUser = okapitoken
 
     * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
     * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
 
     * configure headers = headersUser
+    * configure retry = { count: 4, interval: 1000 }
 
-    # load global variables
     * callonce variables
 
     * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
```

Comment:
Azizbek today showed me that these are now globally defined in our karate-config.json